src/example-types/esTwoLevelAggregation.js

```diff
@@ -30,19 +36,17 @@ module.exports = {
               context.key_data
             )
           ),
-          aggs: {
-            twoLevelAgg: {
-              [context.value_type]: _.omitBy(
-                _.isNil,
-                _.extend(
-                  {
-                    field: context.value_field,
-                  },
-                  context.value_data
-                )
+          aggs: F.arrayToObject(
+            metric =>
+              metric === 'stats' ? 'twoLevelAgg' : `twoLevelAgg_${metric}`,
```

Comment:
I don't see the point in special-casing `stats`. In fact, we should just have `metric` be the key